geth/jail/execution_policy.go

```diff
@@ -32,26 +31,25 @@ func NewExecutionPolicy(
 }
 
 // Execute handles the execution of a RPC request and routes appropriately to either a local or remote ethereum node.
-func (ep *ExecutionPolicy) Execute(req common.RPCCall, call otto.FunctionCall) (*otto.Object, error) {
+func (ep *ExecutionPolicy) Execute(req common.RPCCall, vm *vm.VM) (map[string]interface{}, error) {
 	if params.SendTransactionMethodName == req.Method {
-		return ep.executeSendTransaction(req, call)
+		return ep.executeSendTransaction(vm, req)
 	}
 
 	client := ep.nodeManager.RPCClient()
 
-	return ep.executeWithClient(client, req, call)
+	return ep.executeWithClient(client, vm, req)
```

Comment:
Actually, after moving routing into `rpc` package, there is no need to have `execute with client` function. Either inline its code here, or move `client := ep.nodeManager.RPCClient()` to that function.

Reply:
I guess that make sense, do we still need `ExecutionPolicy`?

Reply:
Can we refactor this out or rename?

Reply:
I think `ExecutionPolicy` refactoring is out of scope. Added to #200.